SSPullToRefresh/SSPullToRefreshView.h

```diff
@@ -105,15 +107,15 @@ typedef NS_ENUM(NSUInteger, SSPullToRefreshViewStyle) {
 
  @see initWithScrollView:delegate:
  */
-@property (nonatomic, assign, readonly) UIScrollView *scrollView;
+@property (nonatomic, weak, readonly, nullable) UIScrollView *scrollView;
```

Comment:
FYI, the change from `assign` to `weak` caused a crash for me:
```
Fatal Exception: NSInternalInconsistencyException
An instance 0x1062ae600 of class UICollectionView was deallocated while key value observers were still registered with it. Current observation info: <NSKeyValueObservationInfo 0x17082c9a0> ( <NSKeyValueObservance 0x171243ea0: Observer: 0x105d80720, Key path: contentOffset, Options: <New: YES, Old: NO, Prior: NO> Context: 0x105d80720, Property: 0x174c42d30> )
```

It occurs on iOS 9 & 10 (not 11), when popping from one VC to another, both with a collection view with a pull to refresh.

Reply:
revert back to `unsafe_unretained` (same as `assign`). this is because of kvo. ref: https://stackoverflow.com/questions/10793206/how-do-you-remove-kvo-from-a-weak-property posted by sam himself

lemme know if this works